liquibase-integration-tests/src/test/resources/changelogs/oracle/complete/root.changelog.xml

```diff
@@ -456,5 +456,17 @@ END testSP;
     <changeSet id="8" author="constraint-check">
         <dropNotNullConstraint tableName="unnamed_constraint" columnName="id"/>
     </changeSet>
+    <changeSet id="SlashEndDelimiterTest" author="mallod">
+        <sql endDelimiter="/" splitStatements="true" stripComments="false">
+            SELECT * FROM unnamed_constraint/
+            update liquibaseRunInfo set timesRan=timesRan+1/
+        </sql>
+    </changeSet>
+    <changeSet id="dividedAndSlashEndDelimiterTest" author="mallod">
+        <sql endDelimiter="/" splitStatements="true" stripComments="false">
+            SELECT a / b FROM unnamed_constraint/
```

Comment:
@MalloD12 I removed previous test, but now it is failing on the divide sign....

Reply:
I'll take a look at it. I thought it could be the spaces between `a` and `b`, but tried with a unit test and that's not the  case.